### ISSUE TITLE
Android build error

### ISSUE THREAD

**Issue by vomchik** [sarriaroman/FabricPlugin#75]
On android build i got this error message

`Error: /Users/vomchik/Projects/LITS/toolook/toolook-app/platforms/android/gradlew: Command failed with exit code 1 Error output:
/Users/vomchik/Projects/LITS/toolook/toolook-app/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));
						^
  symbol: class JavaScriptException
/Users/vomchik/Projects/LITS/toolook/toolook-app/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));
						                             ^
  symbol: class JavaScriptException
`

**Comment by gawainedwards** [comment]
I've also had the same problem:

PATHREMOVED/cordova/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
13:12:43.774 						JavaScriptException ex = new JavaScriptException(data.getString(0));
13:12:43.774 						^
13:12:43.774   symbol: class JavaScriptException
13:12:43.777 PATHREMOVED/cordova/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
13:12:43.777 						JavaScriptException ex = new JavaScriptException(data.getString(0));
13:12:43.777 						                             ^
13:12:43.777   symbol: class JavaScriptException

**Comment by msegers** [comment]
=1

**Comment by msegers** [comment]
It must have something todo with the update 23 hours ago :-)

**Comment by pvdyck** [comment]
confirmed => ionicApp/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));

**Comment by lguardado** [comment]
👍

**Comment by lguardado** [comment]
Is there any way to set the target to 1.1.1 through Travis coniguration? I'm using 'cordova plugin add https://github.com/sarriaroman/FabricPlugin#1.1.1' but it's still getting 1.1.2 version.

**Comment by pvdyck** [comment]
remove the tilde or the carret in front of version number in both package.json and config.xml

**Comment by lguardado** [comment]
Thanks pvdyck, but I don't have anything related to fabric in my package.json or config.xml files... 
I checked the ones in cordova directories too. (The project is integrated with Travis CI)

**Comment by pvdyck** [comment]
in your package.json you should find

 "devDependencies": {
...
    "cordova-fabric-plugin": "1.1.1",
...
},
  "cordovaPlugins": [
"cordova-fabric-plugin"

And in your config.xml

 <plugin name="cordova-fabric-plugin" spec="1.1.1">
    <variable name="FABRIC_API_KEY" value="*key*"/>
    <variable name="FABRIC_API_SECRET" value="*secret*"/>
  </plugin>

**Comment by lguardado** [comment]
Yes, I get what you say, and that's what I was hoping to find... but it isn't there... I think because of the automated process of CI, it runs `cordova plugin add https://github.com/sarriaroman/FabricPlugin#1.1.1` instead of having it in the dependencies

**Comment by pvdyck** [comment]
Exactly... remove it from your CI for now 
=> cordova plugin add https://github.com/sarriaroman/FabricPlugin#1.1.1 might add the tilde in front of the version and install the next minor version

**Comment by lguardado** [comment]
awesome!! thank you so much!! =)

**Comment by lucianomlima** [comment]
+1

**Comment by RoelRoel** [comment]
Why doe this not help in my case? I still get `FabricPlugin.java:142: error: cannot find symbol  JavaScriptException ex = new JavaScriptException(data.getString(0));` while using ` <plugin name="cordova-fabric-plugin" spec="https://github.com/sarriaroman/FabricPlugin#1.1.1">`

**Comment by pvdyck** [comment]
> How can something that does not even build be ever released??

There is no official support from Fabric for this. Therefore, I will not use it anymore.

**Comment by RoelRoel** [comment]
Support of Fabric is not really needed because it is the same native code that is executed via a cordova plugin. The only thing this project does is create a javascript interface for it.

**Comment by pvdyck** [comment]
But you still need someone to do the binding (and obviously test it _before_ releasing it).

**Comment by munavvarfairoos** [comment]
Thank a lot

**Comment by karser** [comment]
+1
The workaround is `"cordova-fabric-plugin": "1.1.1"`

**Comment by zwallacedev** [comment]
@karser are you building this locally or on build.phonegap.com ?

**Comment by karser** [comment]
@zwallacedev I'm building this locally

**Comment by sarriaroman** [comment]
Solved in version 1.1.3

**Comment by lazysoundsystem** [comment]
I've got this error with version 1.1.3 - is there anything else I can try?

```
BUILD FAILED in 2m 45s
34 actionable tasks: 4 executed, 30 up-to-date
(node:136) UnhandledPromiseRejectionWarning: Error: /ionicapp/platforms/android/gradlew: Command failed with exit code 1 Error output:
/ionicapp/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));
						^
  symbol: class JavaScriptException
/ionicapp/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));
```
